README.md

```diff
@@ -132,14 +132,42 @@ To use a different `Typescript TSDK` version than the one that comes with vscode
 
 ## Using Exponentjs
 
-We support using exponentjs to run, debug and publish your applications.
+We support using exponentjs to run, debug and publish your applications. For more information on exponent, see [here](https://docs.getexponent.com/).
+
+To debug a React-Native project in exponent:
+
+0. On a device or emulator, install the [exponent app](https://getexponent.com/)
+0. Ensure that there is no React Native pacakger running
```

Comment:
It doesn't actually matter; the way markdown works is that lines starting with a number end up making a numbered list, but the actual numbers don't matter in general. If you look at https://github.com/Microsoft/vscode-react-native/pull/301/files/31a9209070261d379a04239ef8d1d89e9e2b8907..cab9ea363d94083e7eed156034c6a3caf9e54503?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8 you'll see how it renders